requirements/dev.txt

```diff
@@ -4,73 +4,70 @@
-pytest-django==3.8.0
-pytest==4.6.9
-python-slugify==4.0.0
-pytz==2019.3
-pyyaml==5.3
-requests==2.22.0
-scandir==1.10.0
-singledispatch==3.4.0.3
-six==1.14.0
-snowballstemmer==2.0.0
-stevedore==1.31.0
-text-unidecode==1.3
-toml==0.10.0
-tox-battery==0.5.2
-tox==3.14.3
-urllib3==1.25.8
-virtualenv==16.7.9
-wcwidth==0.1.8
-wrapt==1.11.2
-zipp==1.1.0
+appdirs==1.4.3            # via -r requirements/travis.txt, virtualenv
+argparse==1.4.0           # via -r requirements/quality.txt, caniusepython3
+astroid==2.3.3            # via -r requirements/quality.txt, pylint, pylint-celery
+attrs==19.3.0             # via -r requirements/quality.txt, pytest
+backports.functools-lru-cache==1.6.1  # via -r requirements/quality.txt, caniusepython3
+caniusepython3==7.2.0     # via -r requirements/quality.txt
+certifi==2020.4.5.1       # via -r requirements/quality.txt, -r requirements/travis.txt, requests
+chardet==3.0.4            # via -r requirements/quality.txt, -r requirements/travis.txt, requests
+click-log==0.3.2          # via -r requirements/quality.txt, edx-lint
+click==7.1.1              # via -r requirements/pip-tools.txt, -r requirements/quality.txt, click-log, code-annotations, edx-lint, pip-tools
+code-annotations==0.3.3   # via -r requirements/quality.txt
+codecov==2.0.22           # via -r requirements/travis.txt
+coverage==5.1             # via -r requirements/quality.txt, -r requirements/travis.txt, codecov, pytest-cov
+distlib==0.3.0            # via -r requirements/quality.txt, -r requirements/travis.txt, caniusepython3, virtualenv
+django-model-utils==4.0.0  # via -r requirements/quality.txt
+django==2.2.12            # via -r requirements/quality.txt, code-annotations, django-model-utils
+edx-lint==1.4.1           # via -r requirements/quality.txt
+edx-opaque-keys==2.0.2    # via -r requirements/quality.txt
+filelock==3.0.12          # via -r requirements/travis.txt, tox, virtualenv
+idna==2.9                 # via -r requirements/quality.txt, -r requirements/travis.txt, requests
+importlib-metadata==1.6.0  # via -r requirements/quality.txt, -r requirements/travis.txt, importlib-resources, pluggy, pytest, tox, virtualenv
+importlib-resources==1.4.0  # via -r requirements/travis.txt, virtualenv
+isort==4.3.21             # via -r requirements/quality.txt, pylint
+jinja2==2.11.2            # via -r requirements/quality.txt, code-annotations
+lazy-object-proxy==1.4.3  # via -r requirements/quality.txt, astroid
+markupsafe==1.1.1         # via -r requirements/quality.txt, jinja2
+mccabe==0.6.1             # via -r requirements/quality.txt, pylint
+more-itertools==8.2.0     # via -r requirements/quality.txt, pytest
+packaging==20.3           # via -r requirements/quality.txt, -r requirements/travis.txt, caniusepython3, pytest, tox
+pathlib2==2.3.5           # via -r requirements/quality.txt, pytest
+pbr==5.4.5                # via -r requirements/quality.txt, stevedore
+pip-tools==5.0.0          # via -r requirements/pip-tools.txt
```

Comment:
Add in the `constraints` file.
`# pip20+ requires for greater version of pip-tools. But installing 19.3.1 in all our containers and VMs.`
`pip-tools==4.5.1`